authentication/authentication.go

```diff
@@ -159,6 +164,30 @@ func New(domain string, options ...Option) (*Authentication, error) {
 	a.OAuth = (*OAuth)(&a.common)
 	a.Passwordless = (*Passwordless)(&a.common)
 
+	validatorOpts := []idtokenvalidator.Option{}
+
+	if a.http != http.DefaultClient {
+		validatorOpts = append(validatorOpts, idtokenvalidator.WithHTTPClient(a.http))
+	}
+
+	if a.clockTolerance.Seconds() != 0 {
```

Comment:
Do we have a default value for the clock tolerance? Like https://github.com/auth0/Auth0.swift/blob/master/Auth0/Auth0WebAuth.swift#L25

Reply:
We do but not on the authentication client, it's set down in the [ID token validator](https://github.com/auth0/go-auth0/blob/feat/SDK-4331-id-token-verification/internal/idtokenvalidator/idtokenvalidator.go#L56)

Reply:
Great, my bad 🤦🏼‍♀️